modules/machine-health-checks-about.adoc

```diff
@@ -8,13 +8,25 @@
 
 Machine health checks automatically repair unhealthy machines in a particular machine pool.
 
-To monitor machine health, you create a resource to define the configuration for a controller. You set a condition to check for, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector, and a label for the set of machines to monitor.
+To monitor machine health, create a `MachineHealthCheck` (CR) custom resource that includes a label for the set of machines to monitor and a condition to check, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector. 
+
+The controller that observes a `MachineHealthCheck` (CR) custom resource checks for the condition that you defined. If a machine fails the health check, the machine is automatically deleted and a new one is created to take its place. When a machine is deleted, you see a `machine deleted` event. 
```

Comment:
If you have custom resource (CR) defined above, then you can just say `MachineHealthCheck` CR here